### PR TITLE
Removal of Classic service end point support

### PR DESCRIPTION
**Pull request by 20shivangi**

### CODE REVIEW

File: Tasks/AzurePowerShellV4/Utility.ps1 [hunk 1]
```diff
@@ -0,0 +1,114 @@
+$rollForwardTable = @{
+    "5.0.0" = "5.1.1";
+};
+
+function Get-SavedModulePath {
+    [CmdletBinding()]
+    param([string] $azurePowerShellVersion,
+          [switch] $Classic)
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzurePowerShellV4/Utility.ps1]: Will take it  as part of next PR

File: Tasks/AzurePowerShellV4/Tests/Utility.Get-LatestModule.ps1 [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+# Arrange.
+. $PSScriptRoot\..\..\..\Tests\lib\Initialize-Test.ps1
+. $PSScriptRoot\..\Utility.ps1
+
+$azureRmModulePath = "c:\modules\azurerm_4.1.0"
+$azureModulePath = "c:\modules\azure_3.6.0"
+
+$azureRmModulePattern = "^azurerm_[0-9]+\.[0-9]+\.[0-9]+$"
+$azureModulePattern = "^azure_[0-9]+\.[0-9]+\.[0-9]+$"
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzurePowerShellV4/Tests/Utility.Get-LatestModule.ps1]: Will take it as part of next PR

File: Tasks/AzurePowerShellV4/AzurePowerShell.ps1 [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+try {
+    # Initialize Azure.
+    Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
+    if (Get-Module Az.Profile -ListAvailable){
+         Initialize-AzModule -Endpoint $endpoint
+    }
+    else{
+         Initialize-AzureRMModule -Endpoint $endpoint
+    }
```
[the remark this answers is not in the document]
Comment (20shivangi) [reply Tasks/AzurePowerShellV4/AzurePowerShell.ps1]: Will take it in next PR

File: Tasks/AzurePowerShellV4/AzurePowerShell.ps1 [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+   $error = $_.Exception.Message
+   Write-Verbose "Unable to get the authScheme $error" 
+}
+
+Update-PSModulePathForHostedAgent -targetAzurePs $targetAzurePs -authScheme $authScheme
+
+try {
+    # Initialize Azure.
+    Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
+    if (Get-Module Az.Profile -ListAvailable){
```
Comment (arjgupta) [inline Tasks/AzurePowerShellV4/AzurePowerShell.ps1]: Change az.profile to az.accounts

Comment (arjgupta) [inline Tasks/AzurePowerShellV4/AzurePowerShell.ps1]: fix indentation

**Pull request closed without merging**